src/Groups/directproducts.jl

```diff
@@ -373,6 +489,27 @@ typing
 ```
     W(g_1,...,g_n, h).
 ```
+
+# Examples
+```jldoctest
+julia> G = cyclic_group(3)
+<pc group of size 3 with 1 generators>
+
+julia> H = symmetric_group(2)
+Sym( [ 1 .. 2 ] )
+
+julia> W = wreath_product(G,H)
+<group of size 18 with 2 generators>
+
+julia> a = gens(W)[1]
```

Comment:
This is more efficient in general (same remark applies to other places, too)
```suggestion
julia> a = gen(W,1)
```